app/models/lifehack.rb

```diff
@@ -1,4 +1,12 @@
 class Lifehack < ActiveRecord::Base
   belongs_to :creator, class_name: 'User'
   has_many :reviews
+  validates :title, presence: true
+  before_validation :strip_whitespace
+
+  private
+
+  def strip_whitespace
+    title.strip unless title.nil?
```

Comment:
Great practice to use a helper method like this. You just need to add a model unit test for this as well.